uitools/cpp/Esri/ArcGISRuntime/Toolkit/UtilityNetworkTraceController.cpp

```diff
@@ -30,6 +30,7 @@
 #include <GraphicsOverlayListModel.h>
 #include <IdentifyLayerResult.h>
 #include <Map.h>
+#include <MapQuickView.h>
```

Comment:
I could build the cpp demo app without adding this include

Reply:
Yes, but if the tool is used for widgets, it would complain about missing `QMouseEvent`